.github/workflows/ci.yml

```diff
@@ -66,7 +66,9 @@ jobs:
         uses: scottbrenner/cfn-lint-action@v2
 
       - name: Run verification
-        run: make check
+        run: |
+          echo "::add-matcher::.github/actionlint-matcher.json"
```

Comment:
Based on https://fusectore.dev/2021/11/19/github-action-problem-matchers.html it is recommended to remove the matcher after running the action

Reply:
Yeah, I have added the `actionlint` as a separate step including the matcher being added and removed. However I wanted to keep it simple for now and experiment with not removing the matcher and not separating the `actionlint` into a dedicated step until we are experiencing issues with it. However I'm ok with doing it the right way if you prefer to keep it separately and having the corresponding matcher added/removed.